doc/user_guide/user_guide.md

```diff
@@ -0,0 +1,291 @@
+| `onprem` | `<PORT>`             | -         | `2580`         | Port of the BucketFS Service                                       |
+| `onprem` | `<SERVICE_NAME>`     | yes       | `bfsdefault`   | Name of the BucketFS Service                                       |
+| `onprem` | `<BUCKET_NAME>`      | yes       | `default`      | Name of the Bucket                                                 |
+| `onprem` | `<PATH>`             | -         | (empty / root) | Path inside the Bucket                                             |
+| `onprem` | `<VERIFY>`           | -         | `true`         | Whether to apply TLS security to the connection                    |
+| `saas`   | `<SAAS_ACCOUNT_ID>`  | yes       | -              | Account ID for accessing an SaaS database instance                 |
+| `saas`   | `<SAAS_DATABASE_ID>` | yes       | -              | Database ID of an Exasol SaaS database instance                    |
+| `saas`   | `<SAAS_PAT>`         | yes       | -              | Personal access token for accessing an SaaS database instance      |
+
+### AAF Python Package
+
+The latest version of AAF can be obtained from [pypi](https://pypi.org), see also the [releases on GitHub](https://github.com/exasol/advanced-analytics-framework/releases).
+
+The following command installs the AAF from pypi:
+
+```bash
+pip install exasol-advanced-analytics-framework
+```
+
+### Script Language Containers (SLCs)
+
+Exasol executes User Defined Functions (UDFs) in an isolated Script Language Container (SLCs).
+Running the AAF requires a custom SLC.
+
+#### Build the SLC
+
+If you want to build the AAF SLC then
+
+1. Download the [AAF Sources](https://github.com/exasol/advanced-analytics-framework/) from GitHub
+2. Install poetry
+3. Use the AAF developer commands for building and deploying custom SLCs into Exasol.
+
+```shell
+poetry run nox build_language_container
+```
+
+See [Install the SLC](#install-the-slc).
+
+#### Download a Pre-built SLC
+
+As an alternative to building SLC yourself you can also download a prebuilt AAF SLC from the [AAF releases](https://github.com/exasol/advanced-analytics-framework/releases/latest) on GitHub.
+
+#### Install the SLC
+
+Installing the SLC requires loading the container file into the BucketFS and registering it to the database:
+
+```shell
+SLC_FILE=.slc/exasol_advanced_analytics_framework_container_release.tar.gz
+LANGUAGE_ALIAS=PYTHON3_AAF
+python -m exasol_advanced_analytics_framework.deploy language-container \
+    --dsn <DB_HOST:DB_PORT> \
+    --db-user <DB_USER> \
+    --db-pass <DB_PASSWORD> \
+    --bucketfs-name <BUCKETFS_NAME> \
+    --bucketfs-host <BUCKETFS_HOST> \
+    --bucketfs-port <BUCKETFS_PORT> \
+    --bucketfs-user <BUCKETFS_USER> \
+    --bucketfs-password <BUCKETFS_PASSWORD> \
+    --bucket <BUCKETFS_NAME> \
+    --path-in-bucket <PATH_IN_BUCKET> \
+    --language-alias "$LANGUAGE_ALIAS" \
+    --container-file "$SLC_FILE"
```

Comment:
the slc deployer should be able to download the prebuilt slc automatically from github, see https://github.com/exasol/transformers-extension/blob/f31b7ea91db5e560b6207b6efcba52eddfd4c7ce/exasol_transformers_extension/deployment/te_language_container_deployer.py#L11

--container-file is only fallback for special circumstances

Reply:
Ok, I am sorry to maybe have created some confusion.
Yes: AAF deploy CLI has already been ported to the latest PEC version and is able to download the SLC file from the specified GitHub release.